pkg/placeholders/utils.go

```diff
@@ -120,3 +122,20 @@ func InterpolatePlaceholders(source string) string {
 		}
 	})
 }
+
+// GetBodyFromFileOrInlined returns the correct content for the body of a request.
+// the body of the request can either be inlined, or come from a file
+func GetBodyFromFileOrInlined(source string) (string, error) {
+
+	if strings.HasPrefix(source, "file:") {
```

Comment:
Can we move the `file:` prefix to a constant to avoid hardcoding this value below?